improver_tests/psychrometric_calculations/cloud_condensation_level/test_CloudCondensationLevel.py

```diff
@@ -94,25 +98,32 @@ def metadata_ok(ccl: Cube, baseline: Cube, model_id_attr=None) -> None:
     Raises:
         AssertionError: If anything doesn't match
     """
-    assert ccl.long_name == "air_temperature_at_condensation_level"
-    assert ccl.units == "K"
-    assert ccl.dtype == np.float32
-    ccl.coord("air_pressure")  # Fails if this coord is absent.
-    for coord in ccl.coords():
-        if coord.standard_name == "air_pressure":
-            coord_dims = ccl.coord_dims(coord)
-            assert len(coord_dims) == ccl.ndim
-        else:
+    t_at_ccl, p_at_ccl = ccl
+    assert t_at_ccl.long_name == "air_temperature_at_condensation_level"
+    assert t_at_ccl.units == "K"
+    assert "height" not in [c.name() for c in t_at_ccl.coords()]
+    assert p_at_ccl.long_name == "air_pressure_at_condensation_level"
+    assert p_at_ccl.units == "Pa"
+    assert "height" not in [c.name() for c in p_at_ccl.coords()]
```

Comment:
I thought this might make it harder to follow, while not reducing the number of lines of code, although I could try this:
```suggestion
    assert "height" not in [c.name() for c in p_at_ccl.coords() + t_at_ccl.coords()]
```

Reply:
This works, so committed and pushed.